Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM amazonlinux:2
+
+WORKDIR /usr/src/extension
+
+COPY . .
+
+RUN yum install -y \
+    gzip \
+    tar
+
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash && \
+    . ~/.nvm/nvm.sh && \
+    nvm install node && \
+    npm install yarn -g && \
+    yarn install  && \
+    yarn compile && \
+    yarn lint
+
+ARG BUILD_ID
+
+RUN yum update -y && \
+
+    yum upgrade -y && \
+
+    yum autoremove -y && \
+
+    yum clean all && \
+
+    rm -rf /var/cache/yum
+
+USER nobody
```

Comment:
What is going on in this Dockerfile? If we're just building an artifact I'd have expected a `FROM scratch` at the end.